README.md

```diff
@@ -167,5 +167,5 @@ This example also comes with a robot test file:
 
 ```shell
 cd renode_1.12.0_portable
-test.sh --variable ELF:${PATH_TO_BINARY} ${PATH_TO_REPOSITORY_ROOT}/tests/simple_embedding.robot
+./test.sh --variable BASE_DIR:${PATH_TO_REPOSITORY_ROOT} ${PATH_TO_REPOSITORY_ROOT}/tests/sample_vmvx_sync.robot
```

Comment:
Running `./test.sh` only passes if **all tests** were build for **all libs**. Furthermore, it is expected to find the executables in `build-{cmsis,crt0,libopencm3}`. This is not necessarily the case (when following the README the executables are located in `build`). We should at least add a note on that to this section.

Reply:
True, that was an oversight on my part.
In that case the test will fail 100% when following the README. I don't know if telling a user to run the test is at all helpful then.
An extra testcase just for the README example might be better, but we'd have to separate it so as to prevent it from interfering with the setup for the CI Pipeline.

Reply:
Yes, probably it is best to add a test case that runs `sample_vmvx_sync` (with the stm32f407, not on the stm32f4xx platform!) specifically for the build instructions provided in the README. Maybe we find a clever way to exclude it here https://github.com/iml130/iree-bare-metal-arm/blob/b55fad677bd7d185e0a2936308bd7d429a2760c6/.github/workflows/build-and-test.yml#L277 without the need to list every test that should run.

Reply:
We can use a `GLOBIGNORE` pattern:
```
GLOBIGNORE="tests/do_not_run_in_ci.robot"
${{ env.RENODE }}/test.sh --variable BASE_DIR:$GITHUB_WORKSPACE tests/*.robot
unset GLOBIGNORE
```
_This is only tested locally!_

As an alternative we can use different sub directories for CI and non CI tests.